### ISSUE TITLE
Code fixing, after attempt at adding second wave

### ISSUE THREAD

**Issue by FRiTTERSEiG** [sehsarah/wave-template#3]
Hi I was attempting to add a second wave to my code, and found I was unable to make the second wave work so far. I then tried to take out the pieces of code I had added to my local file, to restore it back to tits default to start again. But have found that the wave will not load now, i will add the code bellow, I thinking that i'm just missing a piece of code or a word I cant see is missing/misspelling. For future thought if you could tell us a better way to revert code back to its origin file that would be grateful. 

var width, height, center;
var points = 10;
var smooth = true;
var path = new Path();
var mousePos = view.center / 2;
var path1Height = mousePos.y;
path1.fillColor = 'black';
initializePath();

function initializePath() {
	center = view.center;
	width = view.size.width;
	height = view.size.height / 2;
	path.segments = [];
	path.add(view.bounds.bottomLeft);
	for (var i = 1; i < points; i++) {
		var point = new Point(width / points * i, center.y);
		path.add(point);
	}
	path.add(view.bounds.bottomRight);
	path.fullySelected = true;
}

function onFrame(event) {
	pathHeight += (center.y - mousePos.y - pathHeight) / 10;
	for (var i = 1; i < points; i++) {
		var sinSeed = event.count + (i + i % 10) * 100;
		var sinHeight = Math.sin(sinSeed / 200) * pathHeight;
		var yPos = Math.sin(sinSeed / 100) * sinHeight + height;
		path.segments[i].point.y = yPos;
	}
	if (smooth)
		path.smooth({ type: 'continuous' });
}

function onMouseMove(event) {
	mousePos = event.point;
}

function onMouseDown(event) {
	smooth = !smooth;
	if (!smooth) {
		// If smooth has been turned off, we need to reset
		// the handles of the path:
		for (var i = 0, l = path.segments.length; i < l; i++) {
			var segment = path.segments[i];
			segment.handleIn = segment.handleOut = null;
		}
	}
}

// Reposition the path whenever the window is resized:
function onResize(event) {
	initializePath();
}

**Comment by sehsarah** [comment]
@FRiTTERSEiG can you add this issue to your repo? and add me to that too @sehsarah 

> way to revert code back to its origin file 

`git reset --hard origin/master` 

This will work to revert your local file to what's last been committed to `master` on Github

**Comment by FRiTTERSEiG** [comment]
Hi, i'm still unsure how to to reset my code to the original master. Possibly it may be that I need to reset it multiple time, but i'm a bit confused. When reset on git bash I was given this piece of code:
$ git reset --hard origin/master
HEAD is now at 74e0917 Update README.md
Future goal is to try get a second wave still. I have attempted multipule different ways, firstly doubling most of the code and creating a Path1 and Path2 function. But seemed to be not working as I still have just a sky blue box. Secondly I tried splitting the initializePath function into multiple functions, by increasing the amount of paths:
var path = new Path();
changed to 
var path = new Path1();
var path = new Path2();
But found that it was similair to what I had attempted in the first place, any help or guidence moving forward would be appreciated, as i'm a tad stuck :/
